modules/win-instance/watchmaker-win-instance.template.cfn.yaml

```diff
@@ -517,6 +560,9 @@ Resources:
                   # Install python
                   & "$BootstrapFile" -PythonUrl "$PythonUrl" -Verbose -ErrorAction Stop
 
+                  # Get the host
```

Comment:
This is already set above. Figuring we got some copy pasta here...

Reply:
def copy pasta, removed